marathon/models/info.py

```diff
@@ -93,12 +97,15 @@ class MarathonZooKeeperConfig(MarathonObject):
     :param str zk:
     :param dict zk_future_timeout:
     :param str zk_hosts:
+    :param str zk_max_versions:
     :param str zk_path:
+    :param str zk_session_timeout:
     :param str zk_state:
     :param int zk_timeout:
     """
 
-    def __init__(self, zk=None, zk_future_timeout=None, zk_hosts=None, zk_path=None, zk_state=None, zk_timeout=None):
+    def __init__(self, zk=None, zk_future_timeout=None, zk_hosts=None, zk_max_versions=None,zk_path=None,
```

Comment:
Sorry if I sound a bit anal, but do you mind adding a space between "zk_max_versions=None,zk_path=None"?